weave/trace_server/clickhouse_trace_server_migrator.py

```diff
@@ -10,18 +10,30 @@
 
 class ClickHouseTraceServerMigrator:
     ch_client: CHClient
+    migration_dir: str
```

Comment:
main change to this file is adding an optional migration_dir field, and passing ins a test_db_migration_alias to apply this allows us to do something like

ClickHouseTraceServerMigrator(ch_client, "costs/migrations")
ClickHouseTraceServerMigrator.apply_migrations(target_db, target_db_migration_alias="costs")

this will then create a different entry in
migrations under cost

Reply:
It seems like a lot of this complexity could be avoided if we simply add the cost migrations to the existing migration directory. Then we don't need to modify any of the migration related scripts and there is a single true version for an entire DB

Reply:
yah that makes sense, I just didnt want to bloat our migration directory with constant cost updates, since i figured we'd want to make frequent cost migrations